src/gpu/intel/jit/gemm/selector/db/kernel.db

```diff
@@ -1200,5 +1200,7 @@ auto _CATALOG_ = kcatalog::toFlatCatalog({
 {{'G', "gemm", {"[FO]", "O", "S"}, {"T", "N", "N"}}, {-1, -1, {-1, -1, -1}, {-1, -1, -1}, {-1, -1, -1}, {-1, -1, -1}, {16, 16, 1}, "ABI"}, "at32+m32@64 am32+m64@64 aB wg 4x8 ca3 ks32 xaf st vav hi pt sr br bk0 sm sn nb 4x0 grf256 sys acb cr16", {16, (LoopType) 255, 256, {(LoopType) 208, (LoopType) 255, (LoopType) 255}, {1048576, 524288, 16777216}, {1048576, 524288, 16777216}, {64, 32, 32}, {4, 8, 1}, 1, (WGType) 1, 257, 24576, 0, {16, 16, 4}, {true, true, true}}, {'W', 1, {1e+06}}},
 {{'G', "gemm", {"[FO]", "O", "S"}, {"T", "N", "N"}}, {-1, -1, {-1, -1, -1}, {-1, 8, -1}, {-1, -1, -1}, {-1, 8, -1}, {16, 16, 1}, "ABIpq"}, "at32 am128 aB wg 2x1x8 ikr xaf st acb hi pt ar sb128 bk0 bm0 nmk sys", {16, (LoopType) 255, 128, {(LoopType) 209, (LoopType) 255, (LoopType) 2}, {16777216, 131072, 16777216}, {262144, 131072, 16777216}, {16, 8, 128}, {2, 1, 8}, 1, (WGType) 0, 4357, 0, 1024, {16, 16, 4}, {true, true, true}}, {'W', 1, {128}}},
-{{'G', "gemm", {"[FO]", "O", "S"}, {"T", "N", "N"}}, {-1, -1, {-1, 9, -1}, {-1, 32, -1}, {-1, 9, -1}, {-1, 32, -1}, {16, 16, 1}, "ABIpq"}, "at32+m128@80 am32+m128@80 aB wg 8x1x4 ikr wx2 xaf acb hi pt ar sb128 bk0 sm sn bm0 nmk sys", {16, (LoopType) 255, 128, {(LoopType) 209, (LoopType) 255, (LoopType) 2}, {16777216, 262144, 16777216}, {262144, 262144, 16777216}, {16, 16, 128}, {8, 1, 4}, 2, (WGType) 1, 4357, 0, 8192, {16, 16, 4}, {true, true, true}}, {'W', 1, {256}}}
+{{'G', "gemm", {"[FO]", "O", "S"}, {"T", "N", "N"}}, {-1, -1, {-1, 9, -1}, {-1, 32, -1}, {-1, 9, -1}, {-1, 32, -1}, {16, 16, 1}, "ABIpq"}, "at32+m128@80 am32+m128@80 aB wg 8x1x4 ikr wx2 xaf acb hi pt ar sb128 bk0 sm sn bm0 nmk sys", {16, (LoopType) 255, 128, {(LoopType) 209, (LoopType) 255, (LoopType) 2}, {16777216, 262144, 16777216}, {262144, 262144, 16777216}, {16, 16, 128}, {8, 1, 4}, 2, (WGType) 1, 4357, 0, 8192, {16, 16, 4}, {true, true, true}}, {'W', 1, {256}}},
+{{'G', "gemm", {"[FO]", "O", "S"}, {"T", "N", "N"}}, {-1, -1, {-1, -1, -1}, {-1, 8, -1}, {-1, -1, -1}, {-1, 8, -1}, {16, 16, 1}, "IAB"}, "at32 am128 aB wg 2x1x8 ikr xaf st acb li ar sb128 bk0 bm0 nmk sys", {16, (LoopType) 255, 128, {(LoopType) 161, (LoopType) 255, (LoopType) 2}, {16777216, 131072, 16777216}, {262144, 131072, 16777216}, {16, 8, 128}, {2, 1, 8}, 1, (WGType) 0, 4357, 0, 1024, {16, 16, 4}, {true, true, true}}, {'W', 1, {128}}},
+{{'G', "gemm", {"[FO]", "O", "S"}, {"T", "N", "N"}}, {-1, -1, {-1, 25, -1}, {-1, 32, -1}, {-1, 25, -1}, {-1, 32, -1}, {16, 16, 1}, "IAB"}, "at32+m128@80 am32+m128@80 aB wg 8x1x4 ikr wx2 xaf acb hi pt ar sb128 bk0 sm sn bm0 nmk sys", {16, (LoopType) 255, 128, {(LoopType) 209, (LoopType) 255, (LoopType) 2}, {16777216, 262144, 16777216}, {262144, 262144, 16777216}, {16, 16, 128}, {8, 1, 4}, 2, (WGType) 1, 4357, 0, 8192, {16, 16, 4}, {true, true, true}}, {'W', 1, {256}}}
```

Comment:
Is this entry a duplicate of line 1203? I didn't spot the difference, except that this one is listed for 25 <= n <= 32, while the other covers 9 <= n <= 32.

Reply:
OK, so when A/B sums are needed you want to use this strategy only for n >= 25?